### ISSUE TITLE
Remove the 2nd part of conflicts

### ISSUE THREAD

**Issue by sstevens2** [carpentries-incubator/git-novice-branch-pr#36]
I think we can replace the part where you see that conflicts "remembers" the resolution.  I don't usually teach it.  Instead maybe we put a callout that mentions the important parts.

**Comment by sstevens2** [comment]
Would like to get other instructors feedback on this.  Do you teach the 2nd part of conflicts?

**Comment by djhoese** [comment]
I taught the whole thing today. By "2nd part" are you referring to the last portion after the conflicts were fixed and merged we then make another edit and merge it?

I'll admit it did feel odd when I was in the middle of that part where I had the thought "wait, what am I trying to show here again?". I think having to explain that "git will try it's best to resolve conflicts for you" is a good enough hand wave for this portion, especially this late in the lesson. I also don't super like the idea of reusing a branch after it is merged. It's not something you (or at least I don't) see in an open source software workflow.

I think it could be removed.

**Comment by sstevens2** [comment]
Yep! that is the part I was thinking about. Where we show it will re-use how the conflict was solved if you merge it the other way.  I agree, I don't like to reuse branches.
